webrtc/webapp/src/main/webapp/WEB-INF/conf/configuration.xml

```diff
@@ -0,0 +1,299 @@
+  
+  This is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of
+  the License, or (at your option) any later version.
+  
+  This software is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+  
+  You should have received a copy of the GNU Lesser General Public
+  License along with this software; if not, write to the Free
+  Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd http://www.exoplatform.org/xml/ns/kernel_1_3.xsd"
+  xmlns="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd">
+  
+  <!-- WebRTC provider plugin -->
+  <external-component-plugins>
+    <target-component>org.exoplatform.webconferencing.WebConferencingService</target-component>
+    <component-plugin>
+      <name>add.clouddriveprovider</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.webconferencing.webrtc.WebrtcProvider</type>
+      <init-params>
+        <properties-param>
+          <name>provider-configuration</name>
+          <property name="disable" value="${webconferencing.webrtc.disable:false}" />
+        </properties-param>
+        <object-param>
+          <name>rtc-configuration</name>
+          <object type="org.exoplatform.webconferencing.webrtc.WebrtcProvider$RTCConfiguration">
+            <!-- field name="bundlePolicy">
+              <string>balanced</string>
+            </field>
+            <field name="iceCandidatePoolSize">
+              <int>0</int>
+            </field>
+            <field name="iceTransportPolicy">
+              <string>all</string>
+            </field>
+            <field name="certificates">
+              <collection type="java.util.ArrayList">
+                <value>
+                  <object type="org.exoplatform.webconferencing.webrtc.WebrtcProvider$RTCCertificate">
+                    <field name="expires">
+                      <long></long>
+                    </field>
+                  </object>
+                </value>
+              </collection>
+            </field -->
+            <field name="iceServers">
+              <collection type="java.util.LinkedHashSet">
+                <value>
+                  <!-- Pool of STUN servers for default use (development & testing) -->
+                  <object type="org.exoplatform.webconferencing.webrtc.WebrtcProvider$ICEServer">
+                    <field name="enabled">
+                      <boolean>${webconferencing.webrtc.default.stun.enabled:true}</boolean>

```

Comment:
STUN servers for development & testing are enabled by default, shouldn't we set prod conf by default (to make sure dev conf will not end in production) ?